Lib/test/test_typing.py

```diff
@@ -2876,6 +2885,10 @@ def test_get_type_hints_ClassVar(self):
                           'x': ClassVar[Optional[B]]})
         self.assertEqual(gth(G), {'lst': ClassVar[List[T]]})
 
+    def test_get_type_hints_wrapped_decoratored_func(self):
+        expects = {'self': ForRefExample}
+        self.assertEqual(gth(ForRefExample.func), expects)
+        self.assertEqual(gth(ForRefExample.nested), expects)
```

Comment:
Two blank lines between top-level definitions:

```suggestion
        self.assertEqual(gth(ForRefExample.nested), expects)

```